packages/coinstac-simulator/src/db-server.js

```diff
@@ -5,58 +5,126 @@
  * @module db-server
  */
 
-require('./handle-errors')();
-const pdbs = require('spawn-pouchdb-server');
-const cp = require('child_process');
-const pdbsConfig = require('./.pouchdb-server-config');
+require('./utils/handle-errors');
+
 const cloneDeep = require('lodash/cloneDeep');
+const common = require('coinstac-common');
+const config = require('./utils/config');
```

Comment:
Eh. I was attempting to log pdbs's output to a file. It was nice to throw it in a _config/local.json_ file. Think that justifies convict-style configuration?